bin/cljfmt.sh

```diff
@@ -1,7 +1,21 @@
 #! /usr/bin/env bash
 
-set -euxo pipefail
+# Reformat only staged files OR all files if --all is passed.
+# If neither of the above are true, will report what needs to happen and do nothing else.
 
-clojure -M:cljfmt \
---indents ./.cljfmt/indents.clj \
-src/metabase/api/collection.clj
+set -euo pipefail
+
+ARG=${1:-}
+
+STAGED_FILES=$(git diff --name-only --cached -- "*.clj" "*.cljc" "*.cljs")
+
+if [ "$ARG" == "--all" ]; then
+  clojure -M:cljfmt \
+    --indents ./.cljfmt/indents.clj
+elif [ "${#STAGED_FILES}" -gt 0 ]; then
+  clojure -M:cljfmt \
+    --indents ./local/indents.clj \
```

Comment:
why does this use a different indents file when there are staged files 😳

Reply:
Fixed, OBE